optimism.tokenlist.json

```diff
@@ -774,6 +774,28 @@
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
     },
+    {
+      "chainId": 1,
+      "address": "0xbC396689893D065F41bc2C6EcbeE5e0085233447",
+      "name": "Perpetual",
+      "symbol": "PERP",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/PERP.png",
```

Comment:
Add this image to the PR with roughly size 256x256 , you can use a smaller version but preferably not larger.

Reply:
The image has added in previous PR (https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/51)

Reply:
Ah yes sorry.